torch/_dynamo/symbolic_convert.py

```diff
@@ -3746,11 +3762,10 @@ def check_inlineable(func):
             ]
             if "_dynamo" not in func.get_filename():
                 hints += [
-                    f"Remove the function `{fn_qualname}` or the file `{func.get_filename()}` "
-                    "from torch/_dynamo/trace_rules.py. More graph breaks may occur as a result of "
-                    "attempting to trace into the function.",
+                    f"Apply `@torch._dynamo.dont_skip_tracing` to the function `{fn_qualname}` "
+                    "to force tracing into the function. "
+                    "More graph breaks may occur as a result of attempting to trace into the function.",
```

Comment:
Do you need a space after "function." ? The other lines have spaces

Reply:
No spaces required at the end of a hint (the other lines need a space since the string is continued on the next line).